### PR TITLE
Fix typo in forgot password flow examples

### PR DESCRIPTION
**Pull request by syed-ahmad**

### CODE REVIEW

File: lib/msal-react/FAQ.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ function Example() {
     const { result, error, login } = useMsalAuthentication(InteractionType.Popup);
 
     useEffect(() => {
-        if (error && error.errorMessage.indexOf("AADB2C90118") > -1) {
+        if (error && error.message.indexOf("AADB2C90118") > -1) {
```
Comment (tnorling) [inline lib/msal-react/FAQ.md]: I don't think this is correct. This error should be of type `ServerError` which has [this shape](https://azuread.github.io/microsoft-authentication-library-for-js/ref/classes/_azure_msal_common.servererror.html). Are you having problems with this?

Comment (syed-ahmad) [reply lib/msal-react/FAQ.md]: Hi @tnorling, yes you are right the error object returned by useMsalAuthentication does have the correct shape `ServerError`. Will update my PR.

Comment (syed-ahmad) [reply lib/msal-react/FAQ.md]: I have updated this PR @tnorling

File: lib/msal-react/FAQ.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ function Example() {
     useEffect(() => {
         const callbackId = instance.addEventCallback((event) => {
             if (event.eventType === EventType.LOGIN_FAILURE) {
-                if (event.error && event.error.errorMessage.indexOf("AADB2C90118") > -1) {
+                if (event.error && event.error.message.indexOf("AADB2C90118") > -1) {
```
Comment (tnorling) [inline lib/msal-react/FAQ.md:136]: Same comment here. Is this causing issues?

**Comment by tnorling** [comment]
Closing as I believe the docs, as they stand today, are correct. Please let me know if you disagree.

**Pull request closed without merging**